packages/native-stack/src/types.tsx

```diff
@@ -682,6 +708,196 @@ export type NativeStackNavigationOptions = {
    */
   unstable_sheetFooter?: () => React.ReactNode;
 };
+interface SharedHeaderBarButtonItem {
+  /**
+   * Title of the button.
+   */
+  title?: string;
+  /**
+   * Style for the button title.
+   */
+  titleStyle?: {
+    fontFamily?: string;
+    fontSize?: number;
+    fontWeight?: string;
+    color?: ColorValue;
+  };
+  /**
+   * Image source for the button
+   */
+  image?: ImageRequireSource;
+  /**
+   * Any SF symbol. Explore them here: https://developer.apple.com/sf-symbols/
+   */
+  sfSymbolName?: string;
+  /**
+   * The style of the item.
+   * "Prominent" only available from iOS 26.0 and later.
+   *
+   * Read more: https://developer.apple.com/documentation/uikit/uibarbuttonitem/style-swift.property
```

Comment:
We should document what each of these values do; a single line is fine.

Linking to Apple docs is ok, but it should be complementary.

Reply:
The thing is they do different things on different iOS. "done" makes the title bold on iOS < 26 but it has the same effect as "prominent" on iOS 26, it gives the liquid glass a background color. I think that documenting this might be difficult because it might become incorrect as soon as Apple makes changes to iOS. I think its better to only reference to the source (Apple). But of course if you disagree I will happily document it 😊